examples/pipelines/hf_dataset_pipeline/components/load_from_hub/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# Add component requirements here
+git+https://github.com/ml6team/express.git@main
```

Comment:
We might have to think about a more efficient way of using this (e.g. if you make local changes in Express and you want to test them). Could you add a TODO or a ticket for it?

Reply:
Yes so @PhilippeMoussalli the examples should always rely on the main branch (not on a certain commit.

I guess we can update the requirements.txt to a certain commit only when creating a new example (for debug purposes), but once it's ready we should update to just the main branch